### DISCUSSION TITLE
欢迎来到 EasyWeChat 社区！

### DISCUSSION THREAD

**Discussion by overtrue** [w7corp/easywechat#2014] [General]
## 👋 欢迎你!
  我们正在使用 GitHub Deiscussion 作为一个地方与我们社区的其他成员联系。我们希望您
  * 提出你想知道的问题。
  * 分享想法。
  * 与其他社区成员互动。
  * 欢迎他人并保持开放的心态。请记住，这是一个我们的社区

  共建💪。

  要想开始，请在下面留言，介绍自己，并告诉我们你在这个社区做什么。

**Comment by TheNorthMemory** [comment]
观察员
